src/client.js

```diff
@@ -268,18 +268,24 @@ Client.prototype.setPassword = function(hash) {
 
 Client.prototype.input = function(data) {
 	var client = this;
-	var text = data.text.trim();
+	var text = data.text.replace(/\s+$/, "");
 	var target = client.find(data.target);
-	if (text.charAt(0) !== "/") {
-		text = "/say " + text;
+
+	// This is either a normal message or a command escaped with a leading '/'
+	if (text.charAt(0) !== "/" || text.charAt(1) === "/") {
+		text = "say " + text.replace(/^\//, "");
+	} else {
+		text = text.substr(1);
 	}
+
 	var args = text.split(" ");
```

Comment:
While testing this, I found an issue: any text starting with one or more whitespace does not send anything nor produce an error. It's because ` test` here becomes `['', 'test']`.
We can fix this with the first of the 2 solutions below. However, I wonder if it is any good or bad to keep multiple whitespaces within commands. For example, `/join   #channel` becomes `['join', '', '', '#channel']` then back to `join   #channel`... I don't know if that could be an issue in some scenarios, in which case the second solution below would solve it.

```js
var args = text.trim().split(" ");
var args = text.split(" ").filter(function (a) { return a.length !== 0; });
```

I might be missing an alternative, but I wanted to address the bug in the first place and look for potential solutions :-)